libs/fairness/src/lib/util/ErrorMetrics.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { localization } from "@responsible-ai/localization";
+
+export interface IErrorOption {
+  key: string;
+  title: string;
+  isMinimization: boolean;
+  description?: string;
+}
+
+export const errorOptions: { [key: string]: IErrorOption } = {
+  disabled: {
+    description: localization.Fairness.Metrics.errorMetricDisabledDescription,
+    isMinimization: false,
+    key: "disabled",
+    title: localization.Fairness.Metrics.errorMetricDisabled
+  },
+  enabled: {
+    description: localization.Fairness.Metrics.errorMetricEnabledDescription,
+    isMinimization: false,
+    key: "enabled",
+    title: localization.Fairness.Metrics.errorMetricEnabled
+  }
+};
+
+export const defaultErrorMetricPrioritization = [
+  // binary classification
+  errorOptions["enabled"].key,
```

Comment:
Perhaps I'm misremembering, but isn't this kind of file used for listing queryable metrics (for performance and fairness metrics)? I would have expected a series of error metrics rather than "enabled" and "disabled".